.github/ISSUE_TEMPLATE/5_performance_speed.yml

```diff
@@ -0,0 +1,191 @@
+      description: |
+        Switch flutter to master channel and run this app on a physical device
+        using profile mode with Skia tracing enabled, as follows:
+        ```console
+        flutter channel master
+        flutter run --profile --trace-skia
+        ```
+        The bleeding edge master channel is encouraged here because Flutter is
+        constantly fixing bugs and improving its performance. Your problem in an
+        older Flutter version may have already been solved in the master channel.
+
+        Can you confirm the issue still persists on the master channel in profile mode?
+      options:
+        - "Yes"
+        - "No"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Timeline Traces
+      description: |
+        Open Flutter DevTools and save a timeline trace of the performance issue
+        so we know which functions might be causing it. See "How to Collect
+        and Read Timeline Traces" on this blog post:
+        https://medium.com/flutter/profiling-flutter-applications-using-the-timeline-a1a434964af3#a499
+        Make sure the performance overlay is turned OFF when recording the trace as that may affect
+        the performance of the profile run. (Pressing ‘P’ on the command line toggles the overlay.)
+
+        If the trace are too large to be uploaded to GitHub, you may upload them as a `zip` file
+        or use online tools like https://pastebin.com to share it.
+      value: |
+        <details><summary>Timeline Traces JSON</summary>
+
+        ```json
+        [Paste the Timeline Traces here]
+        ```
+
+        </details>
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Video demonstration
+      description: |
+        Record a video of the performance issue using another phone so we
+        can have an intuitive understanding of what happened. Don’t use
+        "adb screenrecord", as that affects the performance of the profile run.
+      value: |
+        <details>
+        <summary>Video demonstration</summary>
+
+        [Upload media here]
+
+        </details>
+  - type: dropdown
+    id: target_platforms
+    attributes:
+      label: What target platforms are you seeing this bug on?
+      multiple: true
+      options:
+        - Android
+        - iOS
+        - Web
+        - macOS
+        - Linux
+        - Windows
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: OS/Browser version | Device information
+      description: |
+        Which target OS version is the test system running? For Web, please provide browser version.
+        Please also include the device information (model, OS version, etc).
+    validations:
+      required: true
+  - type: dropdown
+    id: device-kind
+    attributes:
+      label: Does the problem occur on emulator/simulator as well as on physical devices?
+      options:
+        - "Yes"
+        - "No"
```

Comment:
This should be required, and probably have an "Unknown" as a default option. Currently it adds a "None" which doesn't make much sense.

Reply:
There is `None` option there because this field is not required. Switched to `required` field and there will only be two options: `Yes` and `No`.

Reply:
Ya, it's right. I added `Unknown` option.